cli/pkg/release/gb.go

```diff
@@ -23,25 +23,32 @@ func CreateGbPR(version, dir string) (gh.PullRequest, error) {
 
 	branch := "rnmobile/release_" + version
 
-	console.Info("Checking if branch %s exists", branch)
 	exists, _ := gh.SearchBranch("gutenberg", branch)
 
 	if (exists != gh.Branch{}) {
-		console.Info("Branch %s already exists", branch)
-		return pr, nil
+		console.Warn("Branch %s already exists", branch)
+
+		cont := console.Confirm("Do you wish to continue?")
+
+		if !cont {
+			console.Info("Bye 👋")
+			return pr, fmt.Errorf("exiting before creating PR")
+		}
+
 	} else {
 		console.Info("Cloning Gutenberg to %s", dir)
 
 		// Let's clone into the current directory so that the git client can find the .git directory
 		err := git.Clone(repo.GetRepoPath("gutenberg"), "--depth=1", ".")
+
 		if err != nil {
-			return pr, err
+			console.ExitError("There was an error cloning the Gutenberg repository.")
```

Comment:
I don't think we should use `ExitError`  here or in the other changes. The common go practice is to return the error and let the caller decide how to handle it.  I know we "own" both this package and the command packages but still think we should bubble up errors as high as possible before exiting. In this case it would be in the command packages.

What we can do is decorate the error with more context. So in this case something like 
`return pr, fmt.Error("There was an error cloning the Gutenberg repository: %s", err)`

There are also some cases where the err does have some good debugging info. We keep that if all we do is extend the error message with more context.